base/context.go

```diff
@@ -46,6 +47,9 @@ type Context struct {
 	// Required unless Insecure is true.
 	Certs string
 
+	// KeySize is the key size in bits for certificate keys.
+	KeySize int
```

Comment:
We've put all flags in Context. I'd be happy to have a local-only flag. Will poke at it.